lib/salus/report.rb

```diff
@@ -140,7 +140,7 @@ def export_report
                         when 'txt' then to_s(verbose: verbose)
                         when 'json' then to_json
                         when 'yaml' then to_yaml
-                        when 'sarif' then to_sarif
+                        when 'sarif' then to_sarif(directive['sarif_options'] || {})
```

Comment:
this method changed in the request params PR so just watch out for the conflict when we merge 👍